subsys/bluetooth/controller/ll_sw/ull_llcp_common.c

```diff
@@ -235,7 +235,14 @@ static void lp_comm_tx(struct ll_conn *conn, struct proc_ctx *ctx)
 		 * NOTE: As the supervision timeout is at most 32s the normal procedure response
 		 * timeout of 40s will never come into play for the ACL Termination procedure.
 		 */
-		const uint32_t conn_interval_us = conn->lll.interval * CONN_INT_UNIT_US;
+		uint32_t conn_interval_us;
+
+		if (conn->lll.interval >= BT_HCI_LE_INTERVAL_MIN) {
+			conn_interval_us = conn->lll.interval * CONN_INT_UNIT_US;
+		} else {
+			conn_interval_us = (conn->lll.interval + 1U) * CONN_LOW_LAT_INT_UNIT_US;
+		}
```

Comment:
When can `conn->lll.interval` end up being `< BT_HCI_LE_INTERVAL_MIN`?

Reply:
`CONFIG_BT_CONN_PARAM_ANY=y` allows connection interval values less than `BT_HCI_LE_INTERVAL_MIN` and greater than `BT_HCI_LE_INTERVAL_MAX` allowing some vendor implementation to use proprietary connection interval timings represented outside the defined min/max. In the Zephyr Controller, values 0 to 5 are used to stress the scheduling design/implementation for overlapping connection role under babblesim CI testing; a value of 1 respresents 1 ms interval (where in 0 is 500 us, and 5 is 3 ms).

Reply:
I see.

So `conn->lll.interval == 0` is a valid value? 

I count 15 places wher `conn->lll.interval` is multiplied by something else, making all those expression end up as 0. While I believe this PR is fixing one such specific case for a good reason, I have my doubts that this is a proper solution.

Reply:
> There are several places in non-ISO that also multiply by `conn->lll.interval`.

Those are in connection establishment implementation, initiator/central and peripheral connection establishment. Support for initial connection establishment with these non-standard value in not supported/planned or covered in bsim test either. This will have to be a separate task outside the scope of this PR.

> > So conn->lll.interval == 0 is a valid value?
> 
> Missed this question I think :)

yes, Host permits a value of `0` as connection interval when `CONFIG_BT_CONN_PARAM_ANY=y`, but is not supported in Zephyr Controller for connection establishment, only supporting it in LL Connection Update and LL Connection Parameter Request Procedures, refer to c3f557c12b32

Reply:
> This will have to be a separate task outside the scope of this PR.

Will you create a GH issue or something for it then? If we allow `conn->lll.interval == 0` then that seems to be a fairly high risk of being misused as it is not allowed to be `0` by the core spec.

Reply:
Seems its easier to fix the occurrences to help aid future feature development; putting PR back to draft until I get around to fixing it and updating a bsim coverage.